components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div>
+        <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+            {{ $t('modSubmissionForm.healthcareProfessionalId') }}
+        </span>
+        <div class="flex flex-col mt-4">
+            <div class="justify-start items-start flex">
+                <input 
+                    type='text'
+                    placeholder='Search by name or ID to add a doctor'
+                    class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400 text-neutral-600 text-sm font-normal font-['Noto Sans JP'] placeholder-gray-300"
```

Comment:
for inputs, I recommend avoiding fixed pixel widths, try to use a related size or grid, like `w-1/3` for example. this will help it be more responsive across difference resolutions and device sizes

Reply:
This is a preview using the theme colors for the search box:

![image](https://github.com/ourjapanlife/findadoc-web/assets/114712265/0f42b7b9-789d-4689-9a5b-d1a0227328a6)

I read you can also set 'classes'  (in the tailwind.ts file) in tailwind, for example create an input class and use that style on all the input fields. Is this something we can do, since we have so many input fields?